### PR TITLE
ci: Remove Nextflow caching

### PR DESCRIPTION
**Pull request by Emiller88**
Also, just uses the latest release of setup-nextflow.

https://github.com/nf-core/setup-nextflow/issues/19

### CODE REVIEW

File: subworkflows/local/bam_markduplicates/main.nf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 //
+//
```
Comment (maxulysse) [inline subworkflows/local/bam_markduplicates/main.nf:2]: don't forget to remove it

**Comment by maxulysse** [comment]
Looking good, can you update CHANGELOG

File: subworkflows/local/bam_markduplicates/main.nf (hunk 1 above)
Comment (Emiller88) [reply subworkflows/local/bam_markduplicates/main.nf:2]: I'll just remove the whole commit 😉

**Comment by Emiller88** [comment]
My take, we merge this and see if it makes it better or worse, then we'll know if it's setup-nextflow or not. I'm guessing it's all the caching that's going on with the test data as well...

**Comment by maxulysse** [comment]
I don't think it's working as we would want it to work

**Comment by Emiller88** [comment]
I guess we won't know until we try.

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
> I guess we won't know until we try.

Fire away then

**Pull request merged**